src/core/hooks/use-field/use-field.ts

```diff
@@ -63,28 +59,66 @@ export const useField = <T, Err>(
 type FieldState<T, Err> = Atom.Readonly<{
   value: T;
   touched: TouchedValues<T>;
-  errors: FieldErrors<Err>;
-  validating: FieldValidatingState;
+  error: Err | null;
+  isValid: boolean;
+  isValidating: boolean;
 }>;
 
 const createFieldState = <T, Err>(
   state: FormtsAtomState<object, Err>,
-  field: FieldDescriptor<T, Err>
+  field: FieldDescriptor<T, Err>,
+  includeChildrenDependencies: boolean = false
 ): FieldState<T, Err> => {
   const lens = impl(field).__lens;
-
-  return Atom.fuse(
-    (value, touched, errors, validating) => ({
-      value,
-      touched: touched as any,
-      errors,
-      validating,
-    }),
-    Atom.entangle(state.values, lens),
-    Atom.entangle(state.touched, lens),
-    Atom.fuse(x => Helpers.extractFieldErrors(x, field), state.errors),
-    Atom.fuse(x => Helpers.extractFieldValidating(x, field), state.validating)
-  );
+  const path = impl(field).__path;
+
+  return includeChildrenDependencies
+    ? Atom.fuse(
+        (
+          value,
+          touched,
+          error,
+          isValid,
+          isValidating,
+          _childrenErrors,
+          _childrenValidating
+        ) => ({
+          value,
+          touched: touched as any,
+          error,
+          isValid,
+          isValidating,
+          _childrenErrors,
+          _childrenValidating,
```

Comment:
do we need to actually incldue these in the result object? maybe its sufficient that the input will change triggering cretion of new object instance and therefore rerender?